app/electron/main.js

```diff
@@ -196,6 +197,9 @@ app.on('quit', function () {
 const isHeadlessMode = process.argv[1] === 'headless';
 if (isHeadlessMode) {
   serverProcess = startServer(['-html-static-dir', path.join(process.resourcesPath, './build')]);
+  (async () => {
+    await open('http://localhost:4466')
```

Comment:
Please keep the default port in a const variable and then use it here. It will be also useful for when we add the port option.

Another thing: this commit's title mentions that it adds the open package but that's not the main goal of this commit, so it's a better message to write: `desktop: Open the system's defined browser in headless mode` (you already mention the new dependency in the commit's body so it's not needed to repeat in the title).